README.md

```diff
@@ -155,23 +155,25 @@ The compiler has been tested with the following software versions on a Debian ba
  - [GHC][software/ghc], version 8.6.5
  - [Cabal][software/cabal], version 2.4.1.0
  - [Coq][software/coq], versions 8.8 through 8.11
+ - [Agda][software/agda], versions 2.6.1
+ - [Agda Standard Library][software/agda-stdlib], version 1.3
 
-### Base Library
+### Coq Base Library
 
-In order to use the base library, the Coq files in the base library need to be compiled first.
+In order to use the Coq code generated by the Free Compiler, the Coq base library that accompanies the Free Compiler needs to be compiled first.
 Make sure to compile the base library **before installing** the compiler.
 We provide a shell script for the compilation of Coq files.
 To compile the base library with that shell script, run the following command in the root directory of the compiler.
 
 ```bash
-./tool/compile-coq.sh base
+./tool/compile-coq.sh base/coq
 ```
 
-> **Note:** If you add or remove files from the `base` library (or any other directory that contains Coq code that you wish to compile using the script above), the automatically generated Makefile needs to be updated.
+> **Note:** If you add or remove files from the `base/coq` library (or any other directory that contains Coq code that you wish to compile using the script above), the automatically generated Makefile needs to be updated.
 > For this purpose the script provides the command line option `--recompile`.
 >
 > ```bash
-> ./tool/compile-coq.sh --recompile base
+> ./tool/compile-coq.sh --recompile base/coq
 > ```
```

Comment:
Is there anything the user needs to know about the Agda base library and how to install and use it?